DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -157,6 +292,899 @@ def getDateTime(self, uid):
+                         "height": height,
+                         "center_x": -center_x, # for display in Slicer, negate this. 
+                         "center_y": -center_y,  # for display in Slicer, negate this. 
+                         "center_z": center_z
+                        })
+
+      # create and display tableNode 
+      tableNode = self.createBboxTable(poly_infos)
+
+    return poly_infos, tableNode 
+  
+  def extractPointMetadataToVtkTableNode(self, srFileName):
+    """
+    Extracts the point metadata from the SR using highdicom, 
+    and creates a table node. 
+    """
+
+    # read SR using highdicom 
+    sr = hd.sr.srread(srFileName)
+
+    # will store the info needed for table too. 
+    point_infos = [] 
+
+    # First get the planar roi measurement gorups 
+    groups = sr.content.get_planar_roi_measurement_groups()
+
+    for group in groups: 
+
+      # Get the tracking ids 
+      tracking_identifier = group.tracking_identifier
+      tracking_uid = group.tracking_uid
+
+      # Get the findings and finding_sites 
+      finding_type = [group.finding_type.CodeValue, group.finding_type.CodingSchemeDesignator, group.finding_type.CodeMeaning]
+      finding_sites = []
+      for finding_site in group.finding_sites:
+        if hasattr(finding_site, 'ConceptCodeSequence') and finding_site.ConceptCodeSequence:
+          finding_sites.append([finding_site.ConceptCodeSequence[0].CodeValue, 
+                                finding_site.ConceptCodeSequence[0].CodingSchemeDesignator,
+                                finding_site.ConceptCodeSequence[0].CodeMeaning])
+
+      # # list of QualitativeEvaluation 
+      # if (len(group.get_qualitative_evaluations())>0): 
+      #   qual_eval = group.get_qualitative_evaluations()[0] 
+      #   num_qual_eval = len(qual_eval)
+      #   qual_eval_names = [] 
+      #   qual_eval_values = [] 
+      #   for n in range(0,num_qual_eval):
+      #     qual_eval_name_CodeValue = qual_eval[n].name.CodeValue 
+      #     qual_eval_name_CodingSchemeDesignator = qual_eval[n].name.CodingSchemeDesignator 
+      #     qual_eval_name_CodeMeaning = qual_eval[n].name.CodeMeaning 
+      #     qual_eval_value_CodeValue = qual_eval[n].value.CodeValue 
+      #     qual_eval_value_CodingSchemeDesignator = qual_eval[n].value.CodingSchemeDesignator 
+      #     qual_eval_value_CodeMeaning = qual_eval[n].value.CodeMeaning 
+      #     qual_eval_name = [qual_eval_name_CodeValue, 
+      #                       qual_eval_name_CodingSchemeDesignator, 
+      #                       qual_eval_name_CodeMeaning]
+      #     qual_eval_value = [qual_eval_value_CodeValue, 
+      #                        qual_eval_value_CodingSchemeDesignator, 
+      #                        qual_eval_value_CodeMeaning]
+      #     qual_eval_names.append(qual_eval_name)
+      #     qual_eval_values.append(qual_eval_value)
+
+      # findings, clinically significant cancer, ggg 
+      if (len(group)>0): 
```

Comment:
I think this will always be true, an individual group will never be empty

Reply:
True, I fixed this in [branch](https://github.com/deepakri201/QuantitativeReporting/tree/add_planar_annot).